### PR TITLE
Integrate gutenberg-mobile release 1.91.0

### PR DESCRIPTION
**Pull request by SiobhyB**
## Description
This PR incorporates the 1.91.0 release of gutenberg-mobile.
For more information about this release and testing instructions, please see the related Gutenberg-Mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/5564

Release Submission Checklist

- [ ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Review by fluiddot — APPROVED** [review]
LGTM 🎊 !

As on previous GB-mobile releases, before merging this PR we need to wait for the Gutenberg Mobile release PR to be merged first.

**Review by fluiddot — COMMENTED** [review]
Oh, I've just noticed that the UI tests are failing and the errors are related to the editor:

```


WordPressUITests.EditorGutenbergTests
--
  | testAddGalleryBlock, failed: caught error: "timedOut"
  | <unknown>:0
  |  
  | testAddGalleryBlock, XCTAssertEqual failed: ("Content Structure Blocks: 1, Words: 0, Characters: 0") is not equal to ("Content Structure Blocks: 2, Words: 5, Characters: 24") - Unexpected post structure.
  | /usr/local/var/buildkite-agent/builds/MV-MKE-X64-010/automattic/wordpress-ios/WordPress/UITestsFoundation/Screens/Editor/BlockEditorScreen.swift:181
  |  
  | testAddGalleryBlock, XCTAssertEqual failed: ("Content Structure Blocks: 1, Words: 0, Characters: 0") is not equal to ("Content Structure Blocks: 2, Words: 5, Characters: 24") - Unexpected post structure.
  | /usr/local/var/buildkite-agent/builds/MV-MKE-X64-010/automattic/wordpress-ios/WordPress/UITestsFoundation/Screens/Editor/BlockEditorScreen.swift:181
  |  
  | testAddRemoveFeaturedImage, XCTAssertEqual failed: ("Content Structure Blocks: 1, Words: 0, Characters: 0") is not equal to ("Content Structure Blocks: 1, Words: 5, Characters: 24") - Unexpected post structure.
  | /usr/local/var/buildkite-agent/builds/MV-MKE-X64-010/automattic/wordpress-ios/WordPress/UITestsFoundation/Screens/Editor/BlockEditorScreen.swift:181
  |  
  | testAddRemoveFeaturedImage, XCTAssertEqual failed: ("Content Structure Blocks: 1, Words: 0, Characters: 0") is not equal to ("Content Structure Blocks: 1, Words: 5, Characters: 24") - Unexpected post structure.
  | /usr/local/var/buildkite-agent/builds/MV-MKE-X64-010/automattic/wordpress-ios/WordPress/UITestsFoundation/Screens/Editor/BlockEditorScreen.swift:181
  |  
  | testAddRemoveFeaturedImage, XCTAssertEqual failed: ("Content Structure Blocks: 1, Words: 0, Characters: 0") is not equal to ("Content Structure Blocks: 1, Words: 5, Characters: 24") - Unexpected post structure.
  | /usr/local/var/buildkite-agent/builds/MV-MKE-X64-010/automattic/wordpress-ios/WordPress/UITestsFoundation/Screens/Editor/BlockEditorScreen.swift:181
  |  
  | testBasicPostPublishWithCategoryAndTag, XCTAssertEqual failed: ("Content Structure Blocks: 1, Words: 0, Characters: 0") is not equal to ("Content Structure Blocks: 2, Words: 5, Characters: 24") - Unexpected post structure.
  | /usr/local/var/buildkite-agent/builds/MV-MKE-X64-010/automattic/wordpress-ios/WordPress/UITestsFoundation/Screens/Editor/BlockEditorScreen.swift:181
  |  
  | testBasicPostPublishWithCategoryAndTag, XCTAssertEqual failed: ("Content Structure Blocks: 1, Words: 0, Characters: 0") is not equal to ("Content Structure Blocks: 2, Words: 5, Characters: 24") - Unexpected post structure.
  | /usr/local/var/buildkite-agent/builds/MV-MKE-X64-010/automattic/wordpress-ios/WordPress/UITestsFoundation/Screens/Editor/BlockEditorScreen.swift:181
  |  
  | testBasicPostPublishWithCategoryAndTag, XCTAssertEqual failed: ("Content Structure Blocks: 1, Words: 0, Characters: 0") is not equal to ("Content Structure Blocks: 2, Words: 5, Characters: 24") - Unexpected post structure.
  | /usr/local/var/buildkite-agent/builds/MV-MKE-X64-010/automattic/wordpress-ios/WordPress/UITestsFoundation/Screens/Editor/BlockEditorScreen.swift:181
  |  
  | testTextPostPublish, XCTAssertEqual failed: ("Content Structure Blocks: 1, Words: 0, Characters: 0") is not equal to ("Content Structure Blocks: 1, Words: 5, Characters: 24") - Unexpected post structure.
  | /usr/local/var/buildkite-agent/builds/MV-MKE-X64-010/automattic/wordpress-ios/WordPress/UITestsFoundation/Screens/Editor/BlockEditorScreen.swift:181
  |  
  | testTextPostPublish, XCTAssertEqual failed: ("Content Structure Blocks: 1, Words: 0, Characters: 0") is not equal to ("Content Structure Blocks: 1, Words: 5, Characters: 24") - Unexpected post structure.
  | /usr/local/var/buildkite-agent/builds/MV-MKE-X64-010/automattic/wordpress-ios/WordPress/UITestsFoundation/Screens/Editor/BlockEditorScreen.swift:181
  |  
  | testTextPostPublish, XCTAssertEqual failed: ("Content Structure Blocks: 1, Words: 0, Characters: 0") is not equal to ("Content Structure Blocks: 1, Words: 5, Characters: 24") - Unexpected post structure.
  | /usr/local/var/buildkite-agent/builds/MV-MKE-X64-010/automattic/wordpress-ios/WordPress/UITestsFoundation/Screens/Editor/BlockEditorScreen.swift:181
  |  
  |  
  | Executed 19 tests, with 1 test skipped and 12 failures (1 unexpected) in 1587.713 (1587.730) seconds
```

**Comment by derekblank** [comment]
The test failures appear to be caused by the text input having no value:

```
XCTAssertEqual failed: ("Content Structure Blocks: 1, Words: 0, Characters: 0") is not equal to ("Content Structure Blocks: 1, Words: 5, Characters: 24") - Unexpected post structure.
```

The UI test input value has 0 words and 0 characters, and is being compared to the assertion with 5 word and 24 characters. This may be related to an issue with iOS dictation where the text input doesn't "finish" processing the value of the user's input until the input is blurred. This is known and discussed further on this thread:

- https://github.com/wordpress-mobile/gutenberg-mobile/issues/5165#issue-1380183284

**Pull request merged**
